File: Iniciante/1098/1098.c
#include <stdio.h>

int main() {
    int i, j;

    for (i = 0; i <= 10; i += 2) {
        for (j = 1; j <= 3; j++) {
            //tres primeiras linhas
            if (i == 0) {
                printf("I=%d J=%d\n", 0, j);
            } else {
                //tres ultimas linhas
                if (i == 10) {
                    printf("I=%d J=%d\n", 1, j + 1);
                //linhas do meio
                } else {
                    printf("I=%d.%d J=%d.%d\n", 0, i % 10, j, i % 10);
                }
            }   
        }
    }

    
    for (i = 0; i <= 10; i += 2) {
        for (j = 2; j <= 4; j++) {
            //tres primeiras linhas
            if (i == 0) {
                
            } else {
                //tres ultimas linhas
                if (i == 10) {
                    printf("I=%d J=%d\n", 2, j + 1);
                //linhas do meio
                } else {
                    printf("I=%d.%d J=%d.%d\n", 1, i % 10, j, i % 10);
                }
            }   
        }
    }
    
    return 0;
}

File: Iniciante/Pagina 03/1098/1098.py
i = 0
j = 1
for i in range(i, 12, 2):

    #repetiçoes de 3
    for j in range(1, 4):
        #tres primeiras linhas
        if i == 0:
            print("I=%d J=%d" %(0, j))
        else:
            #tres ultimas linhas
            if i == 10:
                print("I=%d J=%d" %(1, j + 1))
            else:
                #linhas do meio
                print("I=%d.%d J=%d.%d" %(0, i % 10, j, i % 10))
    
i = 0
j = 2
for i in range(i, 11, 2):

    #repetiçoes de 3
    for j in range(2, 5):
        #tres primeiras linhas
        if i == 0:
            continue
        else:
            #tres ultimas linhas
            if i == 10:
                print("I=%d J=%d" %(2, j + 1))
            else:
                #linhas do meio
                print("I=%d.%d J=%d.%d" %(1, i % 10, j, i % 10))

